src/Components/IndexScene/ResetFiltersButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Button } from '@grafana/ui';
+
+import { IndexScene } from './IndexScene';
+
+type Props = {
+  indexScene: IndexScene;
+};
+
+export function ResetFiltersButton({ indexScene }: Props) {
+  const { currentFiltersMatchReference } = indexScene.useState();
+
+  return (
+    !currentFiltersMatchReference && (
+      <Button
+        icon="repeat"
+        variant="secondary"
+        onClick={() => indexScene.resetToReferenceQuery()}
+        tooltip="Reset filters to initial values."
```

Comment:
There's just one thing that I noticed while testing this, is that we should be more explicit with the outcome. It will reset the label filters, but it's not being mentioned.

```suggestion
        tooltip="Reset label filters to initial values."
```

<img width="652" height="199" alt="Image" src="https://github.com/user-attachments/assets/cea1452c-3b39-4807-a169-f2df06f4966b" />

Labels reset, but other filters are kept.

<img width="622" height="196" alt="Image" src="https://github.com/user-attachments/assets/4fd6dc62-b921-4a87-a3e5-ad94f5becf42" />

Reply:
If you don't mind, I'll apply the suggestion so we can move forward with this.